### ISSUE TITLE
Is the 32 bit compatible with Windows XP 32 bit?

### ISSUE THREAD

**Issue by luca7766** [muquit/mailsend-go#23]
When I launch the 32 bit version of mailsend-go i get an error "This is not a valid 32 bit application etc...", just testing... on Windows 10 64 the 64 bit version runs OK, I searched for info but I don't see any reference to operating system compatibility, thank you.

**Comment by Montclair** [comment]
Doesn't work for me on XP.  The OS pops up a notice "mailsend-go.exe is not a valid Win32 application."

**Comment by luca7766** [comment]
yeah, not a real problem, just curious to know why, but the author after months didn't reply to this opened issue.

**Comment by muquit** [comment]
The program was cross compiled as per golang instruction:
```
CGO_ENABLED=0 GOOS=windows GOARCH=386 go build -v -o mailsend-go.exe
file mailsend-go.exe
mailsend-go.exe: PE32 executable (console) Intel 80386 (stripped to external PDB), for MS Windows
```
I do not have access to XP. I suspect it has something to do with golang compiler toolchain. It is not working, so I think I will remove win32 support.
Thanks.

**Comment by Wolf-SO** [comment]
>  It is not working, so I think I will remove win32 support.

@muquit maybe I'm mixing things up: if you say "win32 support", do you actually mean "Windows XP support"?
(the 32-bit version runs fine on my Win8.1 machine)

**Comment by ybr1** [comment]
I'm having the same problem. Converted from mailsend.exe (which works but we're told to switch) to mailsend-go.exe and I get "mailsend-go.exe is not a valid Win32 application"
This is on a Windows Terminal System (Windows Server Standard Copyright 2007 32-bit)
Any suggestions?
Thanks in advance!
